master/internal/db/database.go

```diff
@@ -99,6 +99,8 @@ type DB interface {
 	UpdateAllocationState(allocation model.Allocation) error
 	UpdateAllocationStartTime(allocation model.Allocation) error
 	UpdateAllocationProxyAddress(allocation model.Allocation) error
+	AddAllocationAcceleratorData(containerID string, a model.Allocation, nodeName string,
```

Comment:
You can remove this and just not add to this interface

then when you call the method just call it based on db.MethodName and not ```a.db.MethodName```

Reply:
Done!